SFS_binaural_synthesis/ir_generic.m

```diff
@@ -85,7 +85,9 @@
     % If needed interpolate the given impulse response set and weight, delay the
     % impulse for the correct distance
     ir = get_ir(sofa,X,[phi 0],x0(ii,1:3),'cartesian',conf);
-
+    % Correct Green's function amplitude
+    ir = ir/(4*pi);
```

Comment:
Now I am a bit confused, too :) Another try: When a HRTF is loaded, `ir_correct_distance()` applies a weight to it that takes the original measurement distance and the desired distance into account. What we get is the impulse response for a point source exp(-jkr)/r in a certain desired distance r listened to by a dummy head. What is needed is the same impulse responses for a Green's function point source exp(-jkr)/(4pi*r).
Thus, as the factor 1/(4pi) was needed in `dummy_irs()` and for HRTFs and BRIRs as well, we moved it to `ir_generic()` and then `get_ir()` can still be used to load data without this amplitude adjustment.
There might be a problem when we are not doing 2.5D SFS but have line sources as secondary sources, though? I assume this is done in the same way but should not have the factor of 1/(4*pi).